crates/hir_def/src/item_tree.rs

```diff
@@ -116,7 +119,12 @@ impl ItemTree {
     }
 
     fn empty() -> Self {
-        Self { top_level: Default::default(), attrs: Default::default(), data: Default::default() }
+        Self {
+            _c: Count::new(),
```

Comment:
Can we `Default::default` this too?

Reply:
Actually, I wonder if we should just `derive(Default)`, as per style guide?